www/src/scss/edxorg-theme.scss

```diff
@@ -1,3 +1,3 @@
-@import url("https://cdn.jsdelivr.net/npm/@edx/brand-edx.org@alpha/dist/core.min.css");
-@import url("https://cdn.jsdelivr.net/npm/@edx/brand-edx.org@alpha/dist/light.min.css");
+@import url("https://cdn.jsdelivr.net/npm/@edx/elm-theme@1.8.1/dist/core.min.css");
+@import url("https://cdn.jsdelivr.net/npm/@edx/elm-theme@1.8.1/dist/light.min.css");
```

Comment:
As of this comment, 2U no longer has plans to migrate the existing `@edx/brand-edx.org` in favor of cutting MFEs directly to `@edx/elm-theme` once MFEs support tokens-compatible Paragon, so could opt to use `@edx/elm-theme` for the docs site preview now.